src/clr-angular/forms/styles/_containers.clarity.scss

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.containers') {
+    .clr-form-control {
+        margin-top: $clr-forms-baseline * 4;
+        display: flex;
+        flex-direction: column;
+
+        &.clr-form-control-disabled {
```

Comment:
This could be out of the nesting. The styles assigned to this selector look common across forms, no?

Reply:
Unnested, no functional change for the user.